packages/insight/src/components/search.tsx

```diff
@@ -151,22 +171,38 @@ const Search: FC<SearchProps> = ({borderBottom, id, setErrorMessage}) => {
     }, 3000);
   };
 
+  const Pill: FC<{currency?: string, network?: string }> = ({ currency, network }) => {
+    return (
+      pill && currency ?
+        <PillBubble>
+          <img src={`https://bitpay.com/img/icon/currencies/${currency}.svg`} alt={currency} style={{height: '120%'}} />
+          <p style={{textTransform: 'capitalize', color: Black, padding: '5px'}}>{network}</p>
+          <PillCloseButtonCircle onClick={() => setPill(false)}>
+            <img src={CloseLightSvg} style={{height: '100%', padding: '9px'}} />
+          </PillCloseButtonCircle>
+        </PillBubble>
+      : <></>
+    );
+  }
+
   return (
     <>
-      <SearchForm onSubmit={search}>
+      <SearchForm onSubmit={search} borderBottom={borderBottom}>
+        <span style={{display: 'flex', alignItems: 'center' }}>
+        <img src={searchIcon} style={{padding: 7}}></img>
```

Comment:
Please add an indent here as the `span` is the wrapping parent so please indent the children.